threatminer/komand_threatminer/actions/import_hash_report/action.py

```diff
@@ -1,37 +1,41 @@
 import komand
 from .schema import ImportHashReportInput, ImportHashReportOutput
+from komand_threatminer.util import utils
+
 # Custom imports below
-import json
 import requests
 
-class ImportHashReport(komand.Action):
 
+class ImportHashReport(komand.Action):
     API_URL = 'https://www.threatminer.org/imphash.php?api=True&rt=2'
-    
+
     def __init__(self):
         super(self.__class__, self).__init__(
-                name='import_hash_report',
-                description='Fetches information related to a hash',
-                input=ImportHashReportInput(),
-                output=ImportHashReportOutput())
+            name='import_hash_report',
+            description='Fetches information related to a hash',
+            input=ImportHashReportInput(),
+            output=ImportHashReportOutput())
 
     def run(self, params={}):
         query = params.get('query')
```

Comment:
```suggestion
        query = params.get(Input.QUERY')
```
You need to import shema.py if not imported already.